variables.tf

```diff
@@ -272,7 +272,7 @@ variable "cos_kms_key_crn" {
 variable "cos_kms_new_key_name" {
   description = "Name of the Key Protect key to create for encrypting the COS buckets used by the watsonx projects."
   type        = string
-  default     = ""
+  default     = "storage-delegation-key"
```

Comment:
Can I suggest to make this change in a follow up PR? And just use this PR to update the tests?